### PR TITLE
Adding Key concepts and Fabric Create doc

### PR DESCRIPTION
**Pull request by poojaSprophecy**

### CODE REVIEW

**Review by prongs — APPROVED** [review]

File: docs/concepts/concepts.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+| ![Join and split](img/JoinSplit.png)       | [**Join and Split**](/low-code-spark/gems/join-split/)       | The set of Gems that help with the process of merging or splitting DataFrame(s) to create new DataFrame(s) |
+| ![Custom](img/Custom.png)                  | [**Custom**](/low-code-spark/gems/custom/)                   | Set of Gems that do not fall into the other categories.                                                    |
+| ![Subgraph](img/Subgraph.png)              | [**Subgraph**](/low-code-spark/gems/subgraph/)               | A Gem that can contain many other Gems within it.                                                          |
+
+Each Gem _instance_ comes with their own configuration and each produces its own block of output code.
+
+## Jobs
+
+Once you have developed a Spark data Pipeline using Prophecy, you will want to schedule it to run at some frequency. You can schedule just a single Pipeline or create a DAG of Pipelines. You can also add scripts to run in between these Pipelines.
```
Comment (jainabhinav) [inline docs/concepts/concepts.md:75]: Users might not understand DAG directly, let's add an example image below it

File: docs/concepts/fabrics/create_a_fabric.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Using this option, you can create a 14-Day Free Trial Fabric, On Prophecy Managed Databricks. You can use this when trying out Prophecy and when you don't want to connect your own Spark Execution Environment to Prophecy. We already have some sample data and tables created to try out the different functionalities.
+Please refer below video for step-by-step example
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://user-images.githubusercontent.com/121796483/217787623-1cf01df2-54d6-4338-bd59-bd921e101ce9.mp4" title="Databricks Fabric" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+
+In this Fabric you can only change the [Databricks Runtime version](https://docs.databricks.com/runtime/dbr.html#databricks-runtime). The auto-termination timeout, Executor and Driver Machine Type and Job sizes in uneditable.
```
Comment (jainabhinav) [inline docs/concepts/fabrics/create_a_fabric.md:26]:
```suggestion
In this Fabric you can only change the [Databricks Runtime version](https://docs.databricks.com/runtime/dbr.html#databricks-runtime). The auto-termination timeout, Executor and Driver Machine Type and Job sizes are uneditable.
```

File: docs/concepts/fabrics/create_a_fabric.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Please refer below video for step-by-step example
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://user-images.githubusercontent.com/121796483/217735090-41853091-ef2e-4d60-bdf6-62fe31a7ee3b.mp4" title="Databricks Fabric" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+
+- **Databricks Credentials** - Here you will provide your Databricks Workspace URL and the [Personal Access token](https://docs.databricks.com/dev-tools/api/latest/authentication.html#generate-a-personal-access-token)
+- **Cluster Details** - Here you would need to provide the [Databricks Runtime version](https://docs.databricks.com/runtime/dbr.html#databricks-runtime), Executor and Drive Machine Types and Termination Timeout if any. These cluster details will be used when creating a cluster for Interactive and Scheduled Job runs.
```
Comment (jainabhinav) [inline docs/concepts/fabrics/create_a_fabric.md:40]:
```suggestion
- **Cluster Details** - Here you would need to provide the [Databricks Runtime version](https://docs.databricks.com/runtime/dbr.html#databricks-runtime), Executor and Drive Machine Types and Termination Timeout if any. These cluster details will be used when creating a cluster via Prophecy during Interactive development and for job clusters during Scheduled Databricks Job runs.
```

File: docs/concepts/fabrics/create_a_fabric.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- **Job sizes** - By default, you will see a Small Job size pre created. You can edit or add more Job sizes.Here you can provide total number of the Executors and Core and Memory for them.
```
Comment (jainabhinav) [inline docs/concepts/fabrics/create_a_fabric.md:41]: We need to add details around advanced json in job sizes. Most users end up using that only as they need iam/arn instances etc, auto-scaling clusters defined there, for which json is the only way currently

File: docs/concepts/fabrics/create_a_fabric.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+- **Prophecy Library** - These are some Scala and Python libraries written by Prophecy to provide additional functionalities on top of Spark. These need to be installed in you Spark execution environment when you spawn a new Session. These are publicly available on Maven central and Pypi respectively.
```
Comment (jainabhinav) [inline docs/concepts/fabrics/create_a_fabric.md:42]:
```suggestion
- **Prophecy Library** - These are some Scala and Python libraries written by Prophecy to provide additional functionalities on top of Spark. These would get automatically installed in your Spark execution environment when you attach to a cluster/create new cluster. These libraries are also publicly available on Maven central and Pypi respectively.
```

File: docs/concepts/fabrics/create_a_fabric.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+- **Spark Connection** - Here you will provide the Livy URL, Authentication, Spark version and Scala version. If Spark and Scala version are not given, the values are 2.3.0 for Spark and 2.11 for Scala by default.
+
+- **Job sizes** -
+  By default, you will see a Small Job size pre created. You can edit or add more Job sizes. A Job sie consists off
+
+  - Size of the Driver: Driver Core and Memory
+  - Size of the Executor: Core and Memory for each executor
+  - Total number of Executors
+
+- **Prophecy Library** -
```
Comment (jainabhinav) [inline docs/concepts/fabrics/create_a_fabric.md:64]: This information is redundant, we have defined same thing in databricks section as well. We can club the same concepts together and define those once at the top

File: docs/concepts/fabrics/create_a_fabric.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  These are some Scala and Python libraries written by Prophecy to provide additional functionalities on top of Spark. These need to be installed in you Spark execution environment when you spawn a new Session. These are publicly available on Maven central and Pypi respectively.
+
+- **Spark Config** -
+  These are additional [Spark Properties](https://spark.apache.org/docs/latest/configuration.html#available-properties) you can set which are applied at startup of session.
```
Comment (jainabhinav) [inline docs/concepts/fabrics/create_a_fabric.md:68]:
```suggestion
  These are additional [Spark Properties](https://spark.apache.org/docs/latest/configuration.html#available-properties) spark properties which you can set which would be applied at spark session initialisation.
```

File: docs/concepts/pipelines.md [hunk 8]
```diff
@@ -51,7 +51,7 @@ When editing a `Pipeline`, you'll be using the editor shown below.
    6. Spark Version
    7. Visual Language
 2. _View switch_ : Switch between the Visual development and the Generated Code. **Note:** The generated Code will only update if there are no errors detected in the Visual view.
-3. _Fabric Switch_ : Select the [Fabric](./fabric.md) to use during development, start a new or connect to an existing cluster
+3. _Fabric Switch_ : Select the [Fabric](fabrics/fabrics.md) to use during development, start a new or connect to an existing cluster
```
Comment (jainabhinav) [inline docs/concepts/pipelines.md:54]:
```suggestion
3. _Fabric Switch_ : Select the [Fabric](fabrics/fabrics.md) to use during development, start a new cluster or connect to an existing one.
```

**Review by jainabhinav — CHANGES_REQUESTED** [review]
Left few comments. Also why are we committing yarn.lock file?

File: docs/concepts/concepts.md (hunk 1 above)
Comment (poojaSprophecy) [reply docs/concepts/concepts.md:75]: Added the image

File: docs/concepts/fabrics/create_a_fabric.md (hunk 6 above)
Comment (poojaSprophecy) [reply docs/concepts/fabrics/create_a_fabric.md:64]: For now It is okay to have some redundancy. Once we struture the docs better, we will work on removing them

File: docs/concepts/fabrics/create_a_fabric.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+</div></div>
+
+- **Databricks Credentials** - Here you will provide your Databricks Workspace URL and the [Personal Access token](https://docs.databricks.com/dev-tools/api/latest/authentication.html#generate-a-personal-access-token)
+- **Cluster Details** - Here you would need to provide the [Databricks Runtime version](https://docs.databricks.com/runtime/dbr.html#databricks-runtime), Executor and Drive Machine Types and Termination Timeout if any. These cluster details will be used when creating a cluster via Prophecy during Interactive development and for job clusters during Scheduled Databricks Job runs.
+- **Job sizes** - By default, you will see a Small Job size pre created. You can edit or add more Job sizes.Here you can provide total number of the Executors and Core and Memory for them.
+  When you click on edit for a Job size, you will see two options: _Basic_ and _Advanced(Json)_.
+
+![Editing a Job](img/job_size.png)
```
Comment (jainabhinav) [inline docs/concepts/fabrics/create_a_fabric.md:44]:
```suggestion
![Editing a Job size](img/job_size.png)
```

**Review by jainabhinav — APPROVED** [review]
LGTM, left 1 minor comment

**Pull request merged**